bin/scripts/lib/fonts.json

```diff
@@ -70,6 +70,20 @@
       "repoRelease": true,
       "description": "Metrically similar to Arial, pan-European WGL character set, sans serif"
     },
+    {
+      "unpatchedName": "Atkinson Hyperlegible Mono",
+      "licenseId": "OFL-1.1-RFN",
+      "RFN": true,
+      "version": "1.0",
```

Comment:
This is 2.001 if I'm not mistaken?

Reply:
Sorry, I wasn't sure where to find this and couldn't find anything online. I see now the information is available in FontForge.

Reply:
Note to self: I will add some hints to the appropriate readme

Reply:
Done with 041ef9202ccdba9705b9f96a1fc838b79822a93d